Makefile

```diff
@@ -82,23 +81,17 @@ install-test: clean-build clean-pyc ## install the package and test dependencies
 install-develop: clean-build clean-pyc ## install the package in editable mode and dependencies for development
 	pip install -e .[dev]
 
-
 # LINT TARGETS
 
 .PHONY: lint
-lint: ## check style with flake8 and isort
-	invoke lint
-
-lint-docs: ## check docs formatting with doc8 and pydocstyle
-	doc8 . docs/
-	pydocstyle copulas/
+lint:
+	ruff check .
+	ruff format .  --check
```

Comment:
can we call invoke so we only update in one place

Reply:
Fixed:
https://github.com/sdv-dev/Copulas/pull/407/commits/ff5ced11f1da18dc0f70d12373dfad6e048bcfa5